brands/leisure/bowling_alley.json

```diff
@@ -0,0 +1,13 @@
+{
+  "leisure/bowling_alley|Strike Bowling Bar": {
+    "countryCodes": ["au"],
+    "matchTags": ["leisure/bowling_alley"],
```

Comment:
This `matchTags` property can be removed,  it already matches that one.

Reply:
Thanks for that; looking at the other files it looks unnecessary